gengo/gengo.py

```diff
@@ -270,19 +272,19 @@ def get(self, **kwargs):
                             j['file_key'] = 'file_' + k
                             del j['file_path']
 
-            # If any attachments then modify base url to include
-            # private_key and file_data to include attachments as multipart
-            files = []
-            if 'attachments' in post_data:
+            # If any files then modify base url to include
+            # private_key and file_data to include files as multipart
+            tmp_files = []
+            if 'files' in post_data:
```

Comment:
Could you move this `if` clause into the `try` clause?
The original implementation does not close other files if it fails to open one of the files.